modules/serverless-python-function-return-values.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies
+//
+// * /serverless/functions/serverless-developing-python-functions.adoc
+
+[id="serverless-python-function-return-values_{context}"]
+= Python function return values
+
+Functions may return any value supported by https://flask.palletsprojects.com/en/1.1.x/quickstart/#about-responses[Flask] because the invocation framework proxies these values directly to the Flask server.
+
+.Example
+[source,python]
+----
+def main(context: Context):
+    data = { "message": "Howdy!" }
+    headers = { "content-type": "application/json" }
+    body = { "message": "Howdy!" }
+----
+
```

Comment:
I think here we could have just body and keep the return sentence (something like that, was I meant first time):

```
def main(context: Context):
    body = { "message": "Howdy!" }
    headers = { "content-type": "application/json" }
    return body, 200, headers
```